resources/dicts/patrols/general/medcat.json

```diff
@@ -212,5 +212,80 @@
     "max_cats": 6,
     "antagonize_text": "app1 feels hounded, never given a moment's peace, and they hiss at the StarClan cat. The ghost vanishes like a puff of fog in the morning sun, hopefully not to return.",
     "antagonize_fail_text": "app1 feels hounded, never given a moment's peace, and they hiss at the StarClan cat. But when they turn around, there's only a rather upset looking clanmate, there to fetch them to camp to help with an emergency."
+},
+{
+    "patrol_id": "gen_med_medwar_natureofstarclan",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["med_cat", "warrior", "herb", "random_herbs", "no_app"],
```

Comment:
"herb" tag does nothing without also tagging the names of the herbs you'd like them to gather! it seems like you just want them to grab an assortment though, so you can keep "random_herbs" and remove "herb"